### PR TITLE
Reflect name change from balena VPN to cloudlink

### PR DESCRIPTION
**Pull request by jaomaloy**
This change should reflect that balena VPN has been renamed to cloudlink.

### CODE REVIEW

**Review by vipulgupta2048 — COMMENTED** [review]
Thanks for making this PR @jaomaloy much appreciated. To cohesively reflect the changes everywhere in docs, we probably need to make PR's to external sources as well. 

Namely, the masterclass and some instances in the SV API? Would you be willing to take those up in your own time?

Also, a suggestion. Can you add a definition to cloudlink on this page as well: https://github.com/balena-io/docs/blob/8ff36dd67f4302f066a8e62ebb38998d373016fa/pages/learn/welcome/concepts.md#L12

All in all great work, and LGTM

**Comment by jaomaloy** [comment]
Thanks for the review @vipulgupta2048 ! 

> Namely, the masterclass and some instances in the SV API? Would you be willing to take those up in your own time?
Yes, I'll reach out to supervisor folks. Expect some PRs in the masterclasses. :joy: 

> Also, a suggestion. Can you add a definition to cloudlink on this page as well:
Done

File: pages/learn/welcome/concepts.md [hunk 1]
```diff
@@ -32,3 +32,7 @@ A versioned build of the host OS, specific to a fleet and device type.
 ### Base image
 
 A specific combination of a Linux distribution and additional packages, used as the base operating environment for a service.
+
+### Cloudlink
+
+Previously called {{ $names.company.lower }}VPN, Cloudlink is the connection that allows {{ $names.company.lower }} to have to have a secure tunnel to a device. This uses openVPN as an underlying technology to provides us features like ssh access, the public device URL and the web terminal.
```
[the remark this answers is not in the document]
Comment (jaomaloy) [reply pages/learn/welcome/concepts.md]: thanks @lurch !

File: pages/learn/welcome/concepts.md [hunk 2]
```diff
@@ -32,3 +32,7 @@ A versioned build of the host OS, specific to a fleet and device type.
 ### Base image
 
 A specific combination of a Linux distribution and additional packages, used as the base operating environment for a service.
+
+### Cloudlink
+
+Previously called {{ $names.company.lower }}VPN, Cloudlink is the connection that allows {{ $names.company.lower }} to have to have a secure tunnel to a device. This uses openVPN as an underlying technology to provide features like ssh access, the public device URL and the web terminal.
```
Comment (vipulgupta2048) [inline pages/learn/welcome/concepts.md:38]:
```suggestion
Previously called {{ $names.company.lower }}VPN, this is the connection that allows {{ $names.cloud.lower }} to have a secure tunnel to the device. This feature in balenaCloud uses openVPN as an underlying technology to enable other features like SSH access, the public device URL, and the web terminal.
```

Nitpick: How about we link all 3 features in the end so someone reading this for the first time can figure what these are.

**Review by vipulgupta2048 — COMMENTED** [review]
Thanks for taking the initiative @jaomaloy Looking forward to those PR's. Provided a suggestion for the definition of cloudlink

**Comment by rhampt** [comment]
@jaomaloy Are we sure that the changes you made to the CLI output for example (cmd usage docs, mis-use, update, process etc) that changed `vpn` -> `cloudlink` *actually* spit out those things now?

**Comment by jaomaloy** [comment]
@rhampt , CLI is a good catch, I'll take a look at it and create an issue if there are possible `vpn` -> `cloudlink` instances

File: pages/learn/welcome/concepts.md (hunk 2 above)
Comment (jaomaloy) [reply pages/learn/welcome/concepts.md]: done, thanks @vipulgupta2048

**Comment by vipulgupta2048** [comment]
@jaomaloy I think if this is good to go, then you can certify your changes and merge this.

**Comment by jaomaloy** [comment]
@balena-ci I self-certify!

**Pull request merged**